intergov/channels/shared_db_channel.py

```diff
@@ -0,0 +1,148 @@
+import base64
+import datetime
+from functools import lru_cache
+
+import requests
+
+from intergov.conf import env
+from intergov.loggers import logging
+
+logger = logging.getLogger(__name__)
+
+
+class SharedDbChannel:
```

Comment:
Well, not "SharedDb"Channel - ALL channels will implement the same API. Ie - the node doesn't know what type of channel it's talking to.

Reply:
good, so wording change and removal of the channel type. We still may need to specify API version though, but it might be an over-thinking already.

Are you comfortable with everything else?

Reply:
For Phase 2, yes, let's ignore API versions. Yep, everything else looks good.